joblib/test/test_memory.py

```diff
@@ -229,8 +229,8 @@ def name_collision(x):
         a(1)
         b(1)
 
-        yield assert_equal, len(w), 1
-        yield assert_true, "collision" in str(w[-1].message)
+        assert len(w) == 1
```

Comment:
Pytest has officially declared them deprecated due to "some issues and disadvatages" as per their changelog. Besides, their parametrization mechanism is really cute..